### PR TITLE
Custom recipients

### PR DESCRIPTION
**Pull request by rgwood**
- Recipients are now specified in the initial call to the `submit` endpoint
- We need to do a little bit of work to extract email addresses from the Slack attachment fields (because Slack converts them to `<mailto:>` addresses, boo)
- Unit tests (in Mocha) have been added for the email address extraction

Also some minor formatting changes because I made the mistake of telling VS Code to reformat handler.js...

### CODE REVIEW

**Comment by rgwood** [comment]
Also added type checking using `@ts-check`: https://blogs.msdn.microsoft.com/typescript/2017/04/27/announcing-typescript-2-3/

File: handler.js [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
 const he = require('he');
 const qs = require('qs');
+const slackHelper = require('./slack-helper');
 
 const ses = new AWS.SES();
 
 const EMAIL_TEMPLATE = Handlebars.compile(process.env.EMAIL_TEMPLATE,
-                                          {noEscape: true});
+  {noEscape: true});
+const EMAIL_SEPARATOR = ', ';
```
Comment (rlisagor) [inline handler.js]: The original code was a bit more liberal - accepting a comma or semicolon, then stripping the extra whitespace. Might be worth keeping it (or something similar) to make it easier for people to specify recipient lists?

File: handler.js [hunk 2]
```diff
@@ -1,20 +1,24 @@
+// @ts-check
```
Comment (rlisagor) [inline handler.js]: Is a `tsconfig.json` file needed to make this work?

File: handler.js [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
 'use strict';
 
 const uuidv4 = require('uuid/v4');
 const request = require('request');
+// @ts-ignore aws-sdk is already installed on AWS. Not installing locally to keep the buidl artifacts small
```
Comment (rlisagor) [inline handler.js]: Minor: typo (`buidl` -> `build`)

File: handler.js (hunk 1 above)
Comment (rgwood) [reply handler.js]: Good call, will do.

Comment (rgwood) [reply handler.js]: This separator is only used internally by `ahv-council-thing`. The list of email addresses is first sent to the `submit` function as an array of strings – then the `submit` function joins those into a string for Slack, and the `approve` function needs to split them again.

Given that this is entirely internal to `ahv-council-thing` I think we're OK with a simple approach, let me know if you disagree.

File: handler.js (hunk 2 above)
Comment (rgwood) [reply handler.js]: Done – good catch.

File: handler.js (hunk 1 above)
Comment (rlisagor) [reply handler.js]: Ah yes, that makes sense. I missed the fact that it was in `approve`, not `submit`.

**Pull request merged**